models/api_host.go

```diff
@@ -57,6 +59,7 @@ func (h *Host) ConvertNMHostToAPI() *ApiHost {
 	a.Version = h.Version
 	a.IsDefault = h.IsDefault
 	a.NatType = h.NatType
+	a.PersistentKeepalive = int64(h.PersistentKeepalive)
```

Comment:
divide by `time.Second` for the UI

Reply:
https://github.com/gravitl/netmaker/pull/2604/commits/4828648a8a07d4f0bfd5ce85c56c0e3afc3bfe98